packages/ui/src/working-groups/hooks/useWorkingGroup.ts

```diff
@@ -13,6 +17,14 @@ export function useCountOpenings(groupId: string) {
   return { isLoading: loading, openings: data?.workingGroupOpeningsConnection.totalCount }
 }
 
+export function useCountWorkers(groupId: string) {
+  const { data, loading } = useCountWorkingGroupWorkersQuery({
+    variables: { groupId_eq: groupId, status_json: { isTypeOf_eq: WorkerStatusTypename['active'] } },
+  })
+
+  return { isLoading: loading, workers: data?.workersConnection.totalCount }
+}
```

Comment:
This should go to its own file `useCountWorkers.ts`

Reply:
✅